MeesGame/Gameplay/GameObjects/Player.cs

```diff
@@ -145,6 +154,16 @@ public override void HandleInput(InputHelper inputHelper)
                 nextAction = WEST;
             }
         }
+
+        public bool PlayerHasKey()
```

Comment:
Ik zou dit hernoemen naar HasKey aangezien het een functie is van de Player klasse